### PR TITLE
New shop search

### PR DESCRIPTION
**Pull request by architaharathi**
### Description
Redos the shopsearch page (without the toolbar/navbar components)

### Context
Matching the new designs

### Issue
#149 

### Screenshot
Simulator Screen Shot - iPhone XR - 2019-04-20 at 15 45 01

### CODE REVIEW

**Comment by architaharathi** [comment]
Simulator Screen Shot - iPhone XR - 2019-04-20 at 15 45 01
added package image!

**Comment by architaharathi** [comment]
here is the right screenshot haha
Simulator Screen Shot - iPhone XR - 2019-04-20 at 15 58 34

File: screens/ShopSearch.js [hunk 1]
```diff
@@ -77,164 +77,174 @@ export default class ShopSearch extends React.Component {
...
+                  <View style={{marginLeft: window.width*.6, marginTop: -window.height*0.03}}>
+                    <DropDown selectedValue={this.state.pickerSelectionSize} setState={this.setPickerSize} items={sizeItems}/>
                   </View>
-                <View style={styles.searchLine}/>
-                <View>
-                <View style={styles.rowBox}>
-                  <Text style={styles.searchText2}>How many packages do you have?</Text>
+                  <View style={styles.searchLine}/>     
+                </View>
+                <View style={styles.whiteBox2}>
```
Comment (ijemmao) [inline screens/ShopSearch.js]: Remember you shouldn't have numbers in your variable names. You already made a `whiteBox` style. Just reuse it.

File: screens/ShopSearch.js [hunk 2]
```diff
@@ -77,164 +77,174 @@ export default class ShopSearch extends React.Component {
...
                   </View>
-                <View style={styles.searchLine}/>
-                <View>
-                <View style={styles.rowBox}>
-                  <Text style={styles.searchText2}>How many packages do you have?</Text>
+                  <View style={styles.searchLine}/>     
+                </View>
+                <View style={styles.whiteBox2}> 
+                  <View style={styles.rowBox}>
+                    <Text style={styles.searchText2}>How many packages do you have?</Text>
```
Comment (ijemmao) [inline screens/ShopSearch.js]: Again, you already made a `searchText`. Reuse styles when possible

File: screens/ShopSearch.js [hunk 3]
```diff
@@ -77,164 +77,174 @@ export default class ShopSearch extends React.Component {
...
+      shadowRadius: 5,
+      alignItems: 'center',
+      justifyContent: 'center',
+      marginTop: window.height*.07,
+      flexDirection: 'column',
+      borderColor: '#19C6D1',
+      borderWidth: 1.5,
+      borderRadius: 5,
+    },
+    whiteBox2: {
```
Comment (ijemmao) [inline screens/ShopSearch.js]: Notice that you just made two styles that are exactly the same. Only use one

File: screens/ShopSearch.js [hunk 4]
```diff
@@ -77,164 +77,174 @@ export default class ShopSearch extends React.Component {
...
+      shadowRadius: 4,
+      marginTop: window.height*.005
+    },
+    searchText: {
+      fontSize: 24,
+      color: '#212121',
+      fontFamily: 'Montserrat-Bold',
+      marginLeft: window.width*.01,
+    },
+    searchText2: {
```
Comment (ijemmao) [inline screens/ShopSearch.js]: Again only use one style.

**Comment by architaharathi** [comment]
just a note, the two texts are different because on has different margins so I cannot use the same styling, otherwise it messes up the page; i fixed the white box repeat though

**Comment by ijemmao** [comment]
@architaharathi This is looking good. Instead of `marginTop` use `marginBottom` that way you don't have to have two styles that are really similar being duplicated.

**Pull request merged**